app/scripts/background.js

```diff
@@ -1358,6 +1381,10 @@ async function initBackground() {
   onNavigateToTab();
   try {
     await initialize();
+
+    // Set the extension to idle state after initialization is complete
+    extensionUpdateManager.setIdleState(true);
```

Comment:
This is already called in `initialize()` itself. Do we need to call it twice? If not, I think it is even better to cal;l it done here in the `initBackground` function.

Reply:
Good catch ! Addressed in 316d7758f9a8c900c988fc3b6839028ddc60c2db